llpc/test/shaderdb/extensions/OpExtInst_TestInterpolateAtCentroidNoPersp_lit.frag

```diff
@@ -0,0 +1,32 @@
+#version 450
+
+layout(location = 0) in noperspective centroid float f1_1;
+layout(location = 1) in vec4 f4_1;
+
+layout(location = 0) out vec4 fragColor;
+
+void main()
+{
+    float f1_0 = interpolateAtCentroid(f1_1);
+
+    vec4 f4_0 = interpolateAtCentroid(f4_1);
+
+    fragColor = (f4_0.y == f1_0) ? vec4(0.0) : vec4(1.0);
+}
+// BEGIN_SHADERTEST
```

Comment:
Can you put the checks at the top of the file? Our test suite is not entirely consistent here, but we generally prefer to have RUN lines at the top to be able to quickly tell what the test does.